pkg/connectors/policymanager/clients/policymanager_grpc.go

```diff
@@ -24,6 +24,19 @@ import (
 
 var _ PolicyManager = (*grpcPolicyManager)(nil)
 
+const (
```

Comment:
This file is going to be deleted in the nearest future, so you can just add some general nolint comment instead of fixing lint issues.